salt/modules/zypper.py

```diff
@@ -14,6 +14,7 @@
 
 # Import 3rd-party libs
 # pylint: disable=import-error,redefined-builtin,no-name-in-module
+import rpm
```

Comment:
We need to try/except this and then load this module in `__virtual__` according to the results. Otherwise, this will stacktrace on installations without the `rpm` module installed.